mvc/action-return-types/7.x/WebApiSample/Repositories/ProductsRepository.cs

```diff
@@ -0,0 +1,67 @@
+namespace WebApiSample.Repositories;
+
+using Microsoft.EntityFrameworkCore;
+using WebApiSample.Models;
+
+public class ProductsRepository
```

Comment:
Should we remove the repository? Code was originally written by an opinionated contractor who insisted on repositories. We've since concluded the documents are easier to follow .

https://twitter.com/nickchapsas/status/1560676334803644416

If you create an issue to remove repository, I can remove that in another PR.

Reply:
I would love to remove the repository. I kept it to keep it compatible with what we have before. Let me do this.

Reply:
97% of customers prefer no repository.